Azure.ResourceManager.Core/Resources/Identity.cs

```diff
@@ -9,19 +9,27 @@
 namespace Azure.ResourceManager.Core
 {
     /// <summary>
-    ///     Represents a managed identity
+    /// Represents a managed identity
     /// </summary>
     public class Identity : IEquatable<Identity>
     {
         private const string SystemAssigned = "SystemAssigned";
         private const string UserAssigned = "UserAssigned";
         private const string SystemAndUserAssigned = "SystemAssigned, UserAssigned";
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Identity"/> class with no <see cref="SystemAssignedIdentity"/>  or <see cref="UserAssignedIdentity"/>.
```

Comment:
use the same text as line 29
```suggestion
        /// Initializes a new instance of the <see cref="Identity"/> class.
```